common/commands/src/validator/mixnet/delegators/rewards/vesting_claim_delegator_reward.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::context::SigningClient;
+use clap::Parser;
+use log::info;
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    #[clap(long)]
+    pub identity: String,
```

Comment:
in some places you use `identity` in other `identity_key` - let's keep it consistent

Reply:
Done - fixed in 2 places